### PR TITLE
Add restaurant page with hard-coded info

### PR DESCRIPTION
**Pull request by jyx11011**
image

### CODE REVIEW

File: src/main/webapp/js/restaurant-page.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+
+/**
+ * Initializes the restaurant page based on the id.
+ */
+function init() {
+  const id = window.location.pathname.substring(12); // Remove '/restaurant/'
+  restaurant = {
+    name: "McDonald",
```
Comment (limli) [inline src/main/webapp/js/restaurant-page.js]: nit: single quotes

**Review by limli — DISMISSED** [review]

**Review by lisasgoh — COMMENTED** [review]
will u be adding the deals for the restaurant?

File: src/main/webapp/js/restaurant-page.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  //   .done((restaurant) => {
+        configureRestaurantInfo(restaurant);
+  //    });
+
+  $.ajax('/api/authentication')
+      .done((loginStatus) => {
+        if (!loginStatus.isLoggedIn) {
+          return;
+        }
+
```
Comment (lisasgoh) [inline src/main/webapp/js/restaurant-page.js]: extra line?

File: src/main/webapp/js/restaurant-page.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+/**
+ * Initializes the restaurant page based on the id.
+ */
+function init() {
+  const id = window.location.pathname.substring(12); // Remove '/restaurant/'
+  restaurant = {
+    name: 'McDonald',
+    photoUrl: 'https://d1nqx6es26drid.cloudfront.net/app/uploads/2019/11/05175538/McD_TheToken%C2%AE_1235_RGB.png',
+  }
+  //$.ajax('/api/restaurants/' + id)
```
Comment (lisasgoh) [inline src/main/webapp/js/restaurant-page.js]: formatting for this part is off, maybe u can just delete the commented code for now?

File: src/main/webapp/js/restaurant-page.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
+/**
+ * Configures restaurant name and photo.
+ * @param {Object} restaurant restaurant object
```
Comment (lisasgoh) [inline src/main/webapp/js/restaurant-page.js:3]:
```suggestion
 * @param {Object} restaurant object
```

File: src/main/webapp/js/restaurant-page.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  $.ajax('/api/follows/restaurants/' + restaurantId,
+      {method: 'DELETE'})
+      .done(() => location.reload());
+}
+
+
+/**
+ * Initializes the restaurant page based on the id.
+ */
+function init() {
```
Comment (lisasgoh) [inline src/main/webapp/js/restaurant-page.js]: maybe call it initRestaurant or smth more specific

File: src/main/webapp/js/restaurant-page.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+ * Unfollows a user and reloads the page.
+ * @param {number} restaurantId the id of restaurant to be unfollowed.
+ */
+function unfollow(restaurantId) {
+  $.ajax('/api/follows/restaurants/' + restaurantId,
+      {method: 'DELETE'})
+      .done(() => location.reload());
+}
+
+
```
Comment (lisasgoh) [inline src/main/webapp/js/restaurant-page.js]: extra line

File: src/main/webapp/js/restaurant-page.js (hunk 4 above)
Comment (limli) [reply src/main/webapp/js/restaurant-page.js:3]: the original looks correct, the param is `restaurant`, and the description is `restaurant object`

**Comment by jyx11011** [comment]
yes, will add deals later.

**Review by lisasgoh — DISMISSED** [review]

**Review by SuyashLakhotia — DISMISSED** [review]

**Review by SuyashLakhotia — APPROVED** [review]

**Pull request merged**